go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cloud-gov/s3-broker
 
-go 1.22
+go 1.21
```

Comment:
Not sure, but do you have to update the Go modules when you do this with `go mod tidy`?

Reply:
I thought the same thing, but running it didn't come back with any changes